python/mlc_llm/loader/huggingface_loader.py

```diff
@@ -99,10 +98,7 @@ def __init__(
         check_parameter_usage(extern_param_map, set(self.torch_to_path.keys()))
 
     def load(
-        self,
-        device: Device,
-        preshard_funcs: Dict[str, Callable] = None,
-        smoothquant_funcs: Dict[str, Callable] = None,
+        self, device: Device, preshard_funcs: Dict[str, Callable] = None
```

Comment:
hmm... SmoothQuant was removed. Is it intentional?
`smoothquant_funcs: Dict[str, Callable] = None,`

Reply:
This is paired with ollm PR: https://github.com/octoml/ollm/pull/756
You can find it in the `slm/loader/huggingface_loader.py`.